queries.yaml

```diff
@@ -21,9 +21,14 @@ sources:
 
   planet_osm_polygon:
     - template: planet_osm_polygon.jinja2
+      # landuse and pois start at 4
+      start_zoom: 4
```

Comment:
Historically we had these at 4, but the data (min_zoom) values I think supports to zoom 2. Perhaps zoom 3 is a middle ground?

Reply:
Follow up issue: https://github.com/tilezen/vector-datasource/issues/1382